pkg/security/probe/probe.go

```diff
@@ -1179,6 +1184,7 @@ func NewProbe(config *config.Config, statsdClient statsd.ClientInterface) (*Prob
 		discarderReq:   newDiscarderRequest(),
 		tcPrograms:     make(map[NetDeviceKey]*manager.Probe),
 		statsdClient:   statsdClient,
+		discarderRate:  utils.NewRateLimiter(time.Second, 20),
```

Comment:
Only 20 discarders per second seems pretty small

Reply:
Should we scale the size of the rate limiter based on the CPU count ?